src/lib/zkapp.ts

```diff
@@ -620,16 +640,16 @@ class SmartContract {
     let inTransaction = Mina.currentTransaction.has();
     let inSmartContract = smartContractContext.has();
     if (!inTransaction && !inSmartContract) {
-      // TODO: it's inefficient to return a fresh account update everytime, would be better to return a constant "non-writable" account update,
-      // or even expose the .get() methods independently of any account update (they don't need one)
-      return selfAccountUpdate(this.address, this.nativeToken);
+      // TODO: it's inefficient to return a fresh party everytime, would be better to return a constant "non-writable" party,
+      // or even expose the .get() methods independently of any party (they don't need one)
```

Comment:
I think we accidentally renamed `account update` to `party` here

Reply:
sloppy merge conflict handling, ty!